db/migrate/20170816193809_remove_string_value_from_calculator_constant.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveStringValueFromCalculatorConstant < ActiveRecord::Migration
+  def change
+    remove_column :calculator_constants, :string_value
```

Comment:
@patrickvinograd @aub this migration looks good to me, but do you want to double check? I think for migrations we cannot never be to safe. My understanding is that postgres just makes the column invisible and wouldn't actually remove it unless you do another operation. 

https://stackoverflow.com/questions/15699989/dropping-column-in-postgres-on-a-large-dataset

Reply:
Since there are so few rows I think we can do whatever we need to this table and it would be fine.